exporter/datadogexporter/metadata.yaml

```diff
@@ -9,3 +9,11 @@ status:
   codeowners:
     active: [mx-psi, dineshg13, liustanley, songy23, mackjmr]
     emeritus: [gbbr]
+
+tests:
+  config:
+    hostname: customhostname
+    api:
+      key: aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
+      site: datadoghq.eu
+  expect_consumer_error: true
```

Comment:
https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/cmd/otelcontribcol/exporters_test.go#L366-L367

According to existing testing config in `cmd/otecontribcol` the queue setting and backoff setting are both disabled for some reason, so I think we should keep consistency here? WDYT? @mx-psi @songy23